R/board_rsconnect.R

```diff
@@ -324,6 +324,20 @@ pin_search.pins_board_rsconnect <- function(board, search = NULL, ...) {
   multi_meta(board, paste0(user, "/", name))
 }
 
+
+#' @rdname board_deparse
+#' @export
+board_deparse.pins_board_rsconnect <- function(board, ...) {
+  if (has_name(board, "url")) {
+    server <- board$url
```

Comment:
When will `url` not be present? 

I think this also needs a fix in `rsc_server()` which I'll make momentarily.

Reply:
I thought I got into that situation when I used `auth = "rsconnect"` that had `server_name` but not `url` but now I am having trouble reproducing. Can you confirm that `url` is always there and sufficient?

Reply:
Yeah, I'm pretty sure — `url` comes from `server$url` which comes from `rsc_server()` which calls either `rsc_server_manual()` or `rsc_server_rsconnect()`, and both set the `url`.